README.md

```diff
@@ -14,12 +14,14 @@ Heroku Connect Toolbelt Plugin
 
     heroku connect:auth              - Authenticate a connection to Salesforce
     heroku connect:preauth           - Store a Salesforce authentication for future use
-    heroku connect:export            - Export a mapping configuration json file
-    heroku connect:import            - Import a mapping configuration json file
-    heroku connect:mapping MAPPING   - Returns a status JSON doc for a single mapping
-    heroku connect:setup             - configure a new connection
+    heroku connect:export            - Export a mapping configuration JSON file
+    heroku connect:import            - Import a mapping configuration JSON file
+    heroku connect:info              - Display connection information 
+    heroku connect:mapping:create    - Create a new mapping
+    heroku connect:mapping:delete    - Delete an existing mapping
+    heroku connect:mapping:state     - Returns the state of a mapping
```

Comment:
This would be `heroku connect:mapping:state MAPPING` to indicate that it has a required argument.